### PR TITLE
add eth-weth example to readme

### PR DESCRIPTION
**Pull request by gsteenkamp89**
closes ACX-3033

### CODE REVIEW

File: apps/example/lib/stake.ts [hunk 1]
```diff
@@ -132,22 +132,18 @@ export const StakerContractABI = [
...
-  userAddress: Address,
-  value: bigint,
-): CrossChainAction {
-  return {
-    target: STAKE_CONTRACT.address,
-    callData: generateStakeCallData(userAddress),
-    value,
-  };
-}
-
```
Comment (gsteenkamp89) [inline apps/example/lib/stake.ts:150]: this wasn't being used so I just deleted it

File: packages/sdk/README.md [hunk 2]
```diff
@@ -230,6 +232,118 @@ await client.executeQuote({
 });
 ```
 
+- ### **Example (B)** - _Stake Native ETH on destination chain_
+
+> This follows the logic in the example app.
+
+To implement this feature, we need to bridge either ETH or WETH from our origin chain to our destination chain. Across will always send WETH to the recipient on the destination chain, **unless the recipient is an EOA**. For our use-case, we want to stake native ETH, moreover we want to do this in one step. For this we recommend using Across's MulticallHandler contract (or deploying your own).
```
Comment (gsteenkamp89) [inline packages/sdk/README.md:239]: Please check this information is accurate

**Review by dohaki — COMMENTED** [review]
It does feel a bit verbose now 🤔 I think it might be better to either change the example app to support USDC or rewrite the guide section in the README to mirror only the example app. wdyt?

**Comment by gsteenkamp89** [comment]
> It does feel a bit verbose now 🤔 I think it might be better to either change the example app to support USDC or rewrite the guide section in the README to mirror only the example app. wdyt?

Yeah I agree this is too verbose. The example app uses eth/weth precisely because it can be a confusing implementation since you have to update calldata AND value when building your cross-chain actions. So maybe it _does_ make sense to just use that example int he README?

**Comment by dohaki** [comment]
> > It does feel a bit verbose now 🤔 I think it might be better to either change the example app to support USDC or rewrite the guide section in the README to mirror only the example app. wdyt?
> 
> Yeah I agree this is too verbose. The example app uses eth/weth precisely because it can be a confusing implementation since you have to update calldata AND value when building your cross-chain actions. So maybe it _does_ make sense to just use that example int he README?

Yeah, agree let's do that. In future iterations, we can add other examples as well.

**Pull request merged**